easy2use

```diff
@@ -320,7 +320,7 @@ function do_list {
   then
     pwd
     verbose "Looking for Kubernetes services in namespace ${K8S_NAMESPACE}"
-    source ${CLI_ROOT}/utilities/K8S/get_k8s_ingress.sh | awk '{if (NR!=1) {print "https://"$2}}'
+    source ${CLI_ROOT}/utilities/K8S/get_k8s_ingress.sh | awk '{if (NR!=1) {print "http://"$3}}'
```

Comment:
Could we use `$package` here instead?

Reply:
Since, `list` which will List all deployed services for a given bundle and for k8s it will List Ingresses to services and user/psw.

However, current `./easy2use list Eiffel` provides below results,
`** Easy2Use Target Configuration **
Target Type: Kubernetes
HELM Version: V3
Domainname: mylocalkube
Cluster configfile: /home/infra/.kube/config
Cluster context: minikube
/eiffel-easy2use/bundles/Eiffel
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
https://nginx
`

This doesn't provide the Ingress host entries for the Eiffel Services as expected. So i changed the positional lookup argument from $2 to $3 and it can result below output for the `list` switch in `easy2use`

`** Easy2Use Target Configuration **
Target Type: Kubernetes
HELM Version: V3
Domainname: mylocalkube
Cluster configfile: /home/infra/.kube/config
Cluster context: minikube
/eiffel-easy2use/bundles/Eiffel
https://dummy-er-eiffel.mylocalkube
https://eiffel-backend-allevents-eiffel.mylocalkube
https://eiffel-backend-artifact-eiffel.mylocalkube
https://eiffel-backend-sourcechange-eiffel.mylocalkube
https://eiffel-backend-testexecution-eiffel.mylocalkube
https://eiffel-er-eiffel.mylocalkube
https://eiffel-frontend-eiffel.mylocalkube
https://eiffel-jenkins-eiffel.mylocalkube
https://eiffel-jenkins-fem-eiffel.mylocalkube
https://eiffel-nexus3-eiffel.mylocalkube
https://eiffel-rabbitmq-eiffel.mylocalkube
https://eiffel-remrem-generate-eiffel.mylocalkube
https://eiffel-remrem-publish-eiffel.mylocalkube
https://eiffel-vici-eiffel.mylocalkube
`

Hence, using $package will not provide expected ingress entries for the eiffel services in k8s based deployment.

Reply:
same here it was `$3` in awk command